test/lang/csharp/test.sh

```diff
@@ -19,5 +19,4 @@ cd "$(dirname "$0")/../../.."
 
 cd test/lang/csharp
 
-dotnet test csharp-npgsql4.csproj
-dotnet test csharp-npgsql5.csproj
+dotnet test csharp-npgsql.csproj
```

Comment:
Are we dropping support for Npgsql 4?

Reply:
See the (updated) commit message: npgsql4 still works, but the type of the exception changed and I didn't want to deal with splitting this up.